docs/content/any/project/changelogs/2021-11-30.md

```diff
@@ -0,0 +1,63 @@
+---
```

Comment:
Should add sections for `django-oso`, `flask-oso` and `sqlalchemy-oso` like these:

https://github.com/osohq/oso/blob/99158a8f891e44759fbb14eea73c617195a1fcbc/docs/content/any/project/changelogs/2021-11-10.md?plain=1#L110-L116